assignment.js

```diff
@@ -66,9 +71,12 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+let numbers = [57, 13, 4, 38] // INSERT ANY NUMBERS
+let currentLargestNumber = Math.max(...numbers)
```

Comment:
@carly-ga nice use of the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) (`...`), this is an advanced technique, just make sure you understand how it works if you choose to continue using it in the future